modules/misc/qt/kconfig.nix

```
@@ -47,7 +47,7 @@ in {
           else if t == "bool" then
             "--type bool ${builtins.toJSON v}"
           else
-            toString v;
+            "'${toString v}'";
```

Perhaps the most robust would be to have something like

```suggestion
            lib.escapeShellArg (toString v);
```

That should also handle, e.g., the case where `v` contains `'`.